src/main/java/uk/gov/hmcts/reform/waworkflowapi/controllers/startworkflow/CreateTaskRequest.java

```diff
@@ -1,17 +1,26 @@
 package uk.gov.hmcts.reform.waworkflowapi.controllers.startworkflow;
 
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonInclude.Include;
 import io.swagger.annotations.ApiModelProperty;
 
 import java.util.Objects;
 
 public class CreateTaskRequest {
     @ApiModelProperty(example = "abc1234567890", required = true, notes = "The case id in CCD")
     private final String caseId;
+    @ApiModelProperty(required = true)
     private final Transition transition;
+    @JsonInclude(Include.NON_NULL)
+    @ApiModelProperty(
+        example = "2020-09-05T14:47:01.250542+01:00",
+        notes = "Optional due date for the task that will be created")
+    private final String dueDate;
 
-    public CreateTaskRequest(String caseId, Transition transition) {
+    public CreateTaskRequest(String caseId, Transition transition, String dueDate) {
         this.caseId = caseId;
         this.transition = transition;
+        this.dueDate = dueDate;
     }
 
     public String getCaseId() {
```

Comment:
suggestion: Lombok Getter annotation.

Reply:
We decided not to use Lombok the other day I did say at the time that it would avoid the code coming up in code coverage but after talking about it the decision was not to use Lombok.